app.py

```diff
@@ -26,54 +26,48 @@ def videogame_POST():
     # try:
     if request.is_json:
         # Process the json data from the message body and store in data variable
-
         data = request.get_json()
 
         # Evaluate if key in data set
         if "title" in data.keys() and "platform" in data.keys():
             # Save title and platform dict values into videogame variable
             videogame = VideoGame(data["title"], data["platform"])
 
-            # Generate uuid for each new videogame entry
-            new_uuid = uuid.uuid4()
-
-            # Pass uuid property into VideoGame Class
-            videogame.uuid = new_uuid
+            # Generate and pass uuid property for each new videogame entry
+            data["uuid"] = uuid.uuid4()
+            data["resource-path"] = f'/videogame/{data["uuid"]}'
+            data["status code"] = 201
```

Comment:
https://stackoverflow.com/questions/7824101/return-http-status-code-201-in-flask

Reply:
Considering the recent status code change, how will the client receive a successful status code output?